src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/__tests__/create_index_pattern_wizard.test.js

```diff
@@ -73,10 +67,16 @@ describe('CreateIndexPatternWizard', () => {
       />
     );
 
+    component.setState({
+      isInitiallyLoadingIndices: false,
+      allIndices: [{}],
```

Comment:
We should probably add an index here as the test says `when there are indices`

Reply:
I was thinking the empty object acts as an index. Were you thinking we should add some props to it?